rbac/server/api/search.py

```diff
@@ -118,6 +122,8 @@ def validate_search_payload(search_query):
         return {"errors": "No search_object_types for search received."}
     if "search_input" not in search_query:
         return {"errors": "No search_input string for search received."}
+    if len(search_query["search_input"]) > 255:
```

Comment:
Doesn't throw any errors on the UI, just continually loads.

Reply:
refactored to sue the utility input validation function and throw a 400 error on search_input exceeding 255 characters